### DISCUSSION TITLE
Email replies with Markdown lists of URLs are now Mangled

### DISCUSSION THREAD

**Discussion by westurner** [community/community#118452] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

STORY: Users can reply by email to Issues and PRs with reasonably simple Markdown without needing to attempt to correct the formatting in the web interface.

(GitHub Email replies with Markdown used to work; at launch IIRC. IDK when specifically that changed. It is a regression IMO.)

- Test Cases
  -
    - tc1:
      - Description: Email reply from Gmail without any text formatting
      - Email input:
        ```markdown
        [vscode.dev](http://vscode.dev/) URLs [1] for pygwalker demo notebooks:
        - https://vscode.dev/github/davidgasquez/datalab/blob/main/notebooks/2023-03-14-Pyodide.ipynb
        - https://vscode.dev/github/Kanaries/pygwalker/blob/main/examples/jupyter_demo.ipynb
        
        [1] https://github.com/microsoft/vscode-remote-repositories-github?tab=readme-ov-file#integration-with-vscodedev
        ```
      - GitHub Issues Output:
        https://github.com/Kanaries/pygwalker/issues/81#issuecomment-2045860184

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by westurner** [comment]
- Attached: Screenshot of tc1 showing the mangled output:
  ![Screenshot from 2024-04-09 15-00-34](https://github.com/community/community/assets/50891/f672a697-283e-4c88-8e8e-8d271ba73690)

**Comment by macovsky** [comment]
also basic reply via email hasn't been respecting quoting for a long time already, it's very annoying:

![2024-04-26_10-36_2](https://github.com/community/community/assets/10503/9bf3082f-51c2-41b0-acae-81a7629f548a)

but when I click edit:

![2024-04-26_10-36](https://github.com/community/community/assets/10503/45d5aa0a-6cbb-4f41-a8e9-9ec988e4679f)

preview formats correctly:

![2024-04-26_10-36_1](https://github.com/community/community/assets/10503/2aa81459-c743-4311-b981-252c0a453610)

but editing doesn't change anything

**Comment by tjex** [comment]
Did markdown rendering ever work? [This post](https://github.blog/2011-03-10-reply-to-comments-from-email/) from GitHub in 2011 makes the case that they don't render Markdown from email replies. 

I'm also frustrated by this. When replying via email, it should be a given that the user is going to write in markdown, just as we do in a normal comment. If people are bothering to use email as an interface for comments, then they clearly want to use it seriously. Otherwise no sane person would opt in to receiving that many emails...

I really think there's a strong case to fix this. It also shouldn't be such a huge effort to render as Markdown, when there is already HTML stripping going on.

**Comment by macovsky** [comment]
> Did markdown rendering ever work? [This post](https://github.blog/2011-03-10-reply-to-comments-from-email/) from GitHub in 2011 makes the case that they don't render Markdown from email replies.

markdown never worked indeed. but now they lost quoting which surely used to work before and it makes replying by email almost useless.
